browser/net/decentralized_dns_network_delegate_helper.cc

```diff
@@ -64,10 +64,8 @@ int OnBeforeURLRequest_DecentralizedDnsPreRedirectWork(
       return net::OK;
     }
 
-    service->rpc_controller()->EnsProxyReaderResolveAddress(
+    service->rpc_controller()->EnsProxyReaderGetResolverAddress(
```

Comment:
I feel like we have a problem here. What if `EnsProxyReaderGetResolverAddress` (or, previously, `EnsProxyReaderResolveAddress`) does a quick `return false;`? It seems the callback will never be called and we will stuck with `net::ERR_IO_PENDING`?

Reply:
Thank you! @yrliou same issue with UnstoppableDomainsProxyReaderGetMany, added return there as well, double check it please, thanks

Reply:
LGTM, thank you.